Jenkinsfile

```
@@ -235,10 +323,41 @@ def buildAndUpload(goos, goarch, pkg, fileExtension) {
     def fileName = "jfrog$fileExtension"
 
     build(goos, goarch, pkg, fileName)
-    uploadBinaryToJfrogReleases(pkg, fileName)
+    uploadBinaryToJfrogRepo21(pkg, fileName)
     sh "rm $jfrogCliRepoDir/$fileName"
 }
 
+def distributeCli(version) {
+    stage("Create release bundle") {
+        createReleaseBundle(version)
+    }
+    stage("Distribute to releases") {
+        distributeToReleases(version)
+    }
+}
+
+def createReleaseBundle(version) {
+    withCredentials([
+       string(credentialsId: 'repo21-ecosystem-automation', variable: 'JFROG_CLI_AUTOMATION_ACCESS_TOKEN'),
+       string(credentialsId: 'repo21-url', variable: 'REPO21_URL')
+       ]) {
+        sh """#!/bin/bash
+                builder/jfrog ds rbc jfrog-cli-rb $version --spec=${cliWorkspace}/${repo}/cli-release/cli-rb-spec.json --spec-vars="VERSION=$version" --sign --url $REPO21_URL/distribution/ --access-token=$JFROG_CLI_AUTOMATION_ACCESS_TOKEN
+        """
+    }
+}
+
+def distributeToReleases(version) {
+    withCredentials([
+    string(credentialsId: 'repo21-ecosystem-automation', variable: 'JFROG_CLI_AUTOMATION_ACCESS_TOKEN'),
+    string(credentialsId: 'repo21-url', variable: 'REPO21_URL')
+    ]) {
+        sh """#!/bin/bash
+                builder/jfrog ds rbd jfrog-cli-rb $version --dist-rules=${cliWorkspace}/${repo}/cli-release/releases_distribution_rule.json --url $REPO21_URL/distribution/ --access-token=$JFROG_CLI_AUTOMATION_ACCESS_TOKEN
```

I think this should be shorter:
```suggestion
                builder/jfrog ds rbd jfrog-cli-rb $version --site=releases.jfrog.io --url $REPO21_URL/distribution/ --access-token=$JFROG_CLI_AUTOMATION_ACCESS_TOKEN
```